src/main/java/stringcaculator/domain/Delimeter.java

```diff
@@ -0,0 +1,26 @@
+package stringcaculator.domain;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class Delimeter {
+    private final Input input;
+
+    private Matcher matcher;
+
+    public Delimeter(Input input) {
+        this.input = input;
+    }
+
+    public boolean useCustomDelimeter() {
+        this.matcher = Pattern.compile(DELEMETER_REGEX.CUSTOM.toString()).matcher(input.getValue());
```

Comment:
Pattern 객체를 상수로 치환해도 괜찮을 것 같은데 어떠신가요?
```suggestion
        this.matcher = CUSTOM_REGEX.matcher(input.getValue());
```

Reply:
상수는 어떠한 가공을 하지 않은 숫자나 문자열만을 나타내야 하는 줄 알았는데 아니였군요.
하나 알아갑니다 🔥